__tests__/server.test.js

```diff
@@ -273,8 +273,68 @@ describe('GET', () => {
                 return request(app)
                 .post('/api/articles/nonesense/comments')
                 .send({username: 'butter_bridge', body: 'am body n soul'})
+                .expect(400)
+                .then(({body}) => {
+                    const {msg} = body
+                    expect(msg).toBe('Invalid Input')
+                })
 
         
     });
 
         })})
+        describe('PATCH', () => {
+            describe('/api/articles/:article_id', () => {
+                test('200 - responds with newly updated article', () => {
+                    const newVotes = 5
+
+                    return request(app)
+                    .patch('/api/articles/4')
+                    .send({inc_votes: newVotes})
+                    .expect(200)
+                    .then(({body}) => {
+                        expect(body[0]).toHaveProperty('author', expect.any(String))
+                        expect(body[0]).toHaveProperty('title', expect.any(String))
+                        expect(body[0]).toHaveProperty('article_id', expect.any(Number))
```

Comment:
You know the article id, you should be asserting it to make sure you get the correct information back.